libs/game/scenes.ts

```diff
@@ -160,8 +169,27 @@ namespace scene {
     //% blockId=cameratop block="camera top"
     //% group="Camera"
     //% help=scene/camera-top
+    //% deprecated=true
     export function cameraTop() {
         const scene = game.currentScene();
         return scene.camera.drawOffsetY;
     }
+
+    /**
+     * Returns the specified camera property
+     * @param property The property to get
+     */
+    //% blockId=getcameraproperty block="camera %property"
```

Comment:
slight preference for "$" over "%" in blockdefs (% matches the index of the param and $ matches the name)

Reply:
WOW. TIL. this explains some confusion I had.

Reply:
Always use `$`!!!

`%` makes it much easier to accidentally break localization or APIs. It has caused many regressions in the past, we just can't upgrade the existing APIs because it would change the crowdin strings